### ISSUE TITLE
I use the hunyuanVideo example and return Empty video

### ISSUE THREAD

**Issue by veridone** [kijai/ComfyUI-HunyuanVideoWrapper#377]
I use the example workflow: hyvideo_skyreel_img2vid_example_01.json
download: 
VAE: https://huggingface.co/Kijai/HunyuanVideo_comfy/blob/main/hunyuan_video_vae_bf16.safetensors
Model: https://huggingface.co/Kijai/SkyReels-V1-Hunyuan_comfy/blob/main/skyreels_hunyuan_i2v_bf16.safetensors
llm_model(ComfyUI/models/LLM/llava-llama-3-8b-text-encoder-tokenizer): https://huggingface.co/Kijai/llava-llama-3-8b-text-encoder-tokenizer/tree/main 
clip_model(ComfyUI/models/clip/clip-vit-large-patch14): https://huggingface.co/openai/clip-vit-large-patch14/tree/main

the Result：
![Image](https://github.com/user-attachments/assets/7484e44c-a9bd-4019-b58d-15b098936edd)

Use GPU:A100 and Run Logs:

got prompt
encoded latents shape torch.Size([1, 16, 1, 68, 120])
Loading text encoder model (clipL) from: /home/work/ComfyUI/models/clip/clip-vit-large-patch14
FETCH ComfyRegistry Data: 5/34
Text encoder to dtype: torch.float16
Loading tokenizer (clipL) from: /home/work/ComfyUI/models/clip/clip-vit-large-patch14
Loading text encoder model (llm) from: /home/work/ComfyUI/models/LLM/llava-llama-3-8b-text-encoder-tokenizer
Loading checkpoint shards: 100%|██████████| 4/4 [00:05<00:00,  1.47s/it]
Text encoder to dtype: torch.float16
Loading tokenizer (llm) from: /home/work/ComfyUI/models/LLM/llava-llama-3-8b-text-encoder-tokenizer
FETCH ComfyRegistry Data: 10/34
/home/work/python310/lib/python3.10/site-packages/transformers/modeling_attn_mask_utils.py:284: UserWarning: backend:cudaMallocAsync ignores max_split_size_mb,roundup_power2_divisions, and garbage_collect_threshold. (Triggered internally at ../c10/cuda/CUDAAllocatorConfig.cpp:320.)
  elif not is_tracing and torch.all(attention_mask == 1):
llm prompt attention_mask shape: torch.Size([1, 161]), masked tokens: 31
FETCH ComfyRegistry Data: 15/34
clipL prompt attention_mask shape: torch.Size([1, 77]), masked tokens: 34
model_type FLOW
The config attributes {'use_flow_sigmas': True, 'prediction_type': 'flow_prediction'} were passed to FlowMatchDiscreteScheduler, but are not expected and will be ignored. Please verify your scheduler_config.json configuration file.
Scheduler config: FrozenDict([('num_train_timesteps', 1000), ('flow_shift', 9.0), ('reverse', True), ('solver', 'euler'), ('n_tokens', None), ('_use_default_values', ['n_tokens', 'num_train_timesteps'])])
Using accelerate to load and assign model weights to device...
Requested to load HyVideoModel
FETCH ComfyRegistry Data: 20/34
loaded completely 38530.481915283206 24454.515747070312 True
FETCH ComfyRegistry Data: 25/34
Input (height, width, video_length) = (544, 960, 21)
The config attributes {'reverse': True, 'solver': 'euler'} were passed to DPMSolverMultistepScheduler, but are not expected and will be ignored. Please verify your scheduler_config.json configuration file.
Swapping 20 double blocks and 10 single blocks
FETCH ComfyRegistry Data: 30/34
image_cond_latents shape: torch.Size([1, 16, 1, 68, 120])
image_latents shape: torch.Size([1, 16, 6, 68, 120])
Sampling 21 frames in 6 latents at 960x544 with 20 inference steps
  0%|          | 0/20 [00:00<?, ?it/s]FETCH ComfyRegistry Data [DONE]
[ComfyUI-Manager] default cache updated: https://api.comfy.org/nodes
nightly_channel: 
https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/remote
FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json [DONE]
[ComfyUI-Manager] All startup tasks have been completed.
100%|██████████| 20/20 [03:06<00:00,  9.34s/it]
Allocated memory: memory=8.601 GB
Max allocated memory: max_memory=11.103 GB
Max reserved memory: max_reserved=11.812 GB
Decoding rows: 100%|██████████| 4/4 [00:03<00:00,  1.13it/s]
Blending tiles: 100%|██████████| 4/4 [00:00<00:00, 10.01it/s]
/home/work/ComfyUI/custom_nodes/ComfyUI-VideoHelperSuite/videohelpersuite/nodes.py:104: RuntimeWarning: invalid value encountered in cast
  return tensor_to_int(tensor, 8).astype(np.uint8)
[VideoHelperSuite] - WARNING - Missing input for pix_fmt has been set to yuv420p
[VideoHelperSuite] - WARNING - Missing input for crf has been set to 19
[VideoHelperSuite] - WARNING - Missing input for save_metadata has been set to True
[VideoHelperSuite] - WARNING - Missing input for trim_to_audio has been set to False
[VideoHelperSuite] - WARNING - Missing input for pix_fmt has been set to yuv420p
[VideoHelperSuite] - WARNING - Missing input for crf has been set to 19
[VideoHelperSuite] - WARNING - Missing input for save_metadata has been set to True
[VideoHelperSuite] - WARNING - Missing input for trim_to_audio has been set to False
Prompt executed in 275.01 seconds

**Comment by jtsanborn1** [comment]
Did you fix it? how?
